packages/ui/src/snackbar/SnackbarContent.tsx

```diff
@@ -33,14 +38,21 @@ const useStyles = makeStyles<
     root: {
       color: Color.White,
       backgroundColor: Color.Dark500,
+      alignItems: 'flex-start',
+      padding: '10px 16px',
+      [theme.breakpoints.up('md')]: {
+        width: 'auto',
+        maxWidth: '512px',
+        minWidth: '432px',
```

Comment:
@TravnikovDev What was maxWidth before? Has the design team approved these changes?

Reply:
Before it was '432px' 
We just discussed it with Nodir p2p. We'll try to find official statement

Reply:
@TravnikovDev Great. If Nodir is aware, everything is ok.